src/Presentation/DonationConfirmationPageSelector.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Presentation;
+
+/**
+ * Selects a random confirmation page from given options
+ *
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class DonationConfirmationPageSelector {
+
+	private $config;
+
+	public function __construct( array $config ) {
+		$this->config = $config;
+	}
+
+	public function selectPage(): string {
+		return empty( $this->config ) ? '' : $this->config[$this->getRandomIndex()];
+	}
+
+	private function getRandomIndex() {
```

Comment:
`array_rand()` is stated as being "not so random". But right, actually in this case "not so random" should still be random enough.